### ISSUE TITLE
alaska sea ice

### ISSUE THREAD

**Issue by shospital** [shospital/alaska-seaice#1]

**Comment by shospital** [comment]
Ebett's comments
Alaskan Arctic page

- [x] 1) Under Sea ice extent summary: change "The labels (column names) used includes:"

- [ ] 2) "Past" - should this historical reference be the same baseline as the SST shiny?  (1985-2015)? If we want to provide or plot anomalies, it would be good to be consistent in the baseline....? Or should it match the median years in Rick's sea ice (1991-2020 - or year before previous)?

- [ ] 3) The paragraph:

"The Alaska region is made up of 6 distinct large marine ecosystems across  (fisheries management): the Gulf of Alaska (GOA), Aleutian Islands (AI), eastern Bering Sea (EBS), northern Bering Sea, and Chukchi Sea and Beaufort Seas (referred to here as the Alaskan Arctic). Fishery management areas include the Gulf of Alaska and Bering Sea/Aleutian Islands. No federal fisheries currently occur in the Alaskan Arctic."

...something like that? There are various ways people split up and define ecosystems in Alaska, which is why I suggested not specifically numbering them. But the Fishery Management Plan (FMP) areas are codified.

4) Data table: if we want to calculate anomalies (see #2), we could add that column to the table maybe?
5) Concentration Data: did you have a chance to play around with merging both the near-real-time and the science quality datasets?  
6) This text is italicized on some pages and not on others:
"Data : Sea Ice Concentration, NOAA/NSIDC Climate Data Record V4, Northern Hemisphere, 25km, Science Quality, 1978-Present, Monthly"
7) Resources - what is the link for "Council Report?" - is that a placeholder that could link to the current year Ecosystem Status Report? (oops - now I see your note under To-do). Might be good to link to [this AFSC ESR landing page](https://www.fisheries.noaa.gov/alaska/ecosystems/ecosystem-status-reports-gulf-alaska-bering-sea-and-aleutian-islands) and eventually to an AKFIN ESR page (in baby stages of development).

Overall tabs: I can't remember exactly where we landed at our last meeting, but I think we could have a tab for "Eastern Bering" and on that page could include the data for southeastern (<60N) and northern (>60N) data in the plot and data table? In terms of fishery management areas, the entire Eastern Bering Sea is managed as one area, but I could envision end-users might be interested in southeastern versus northern data subsets.
